apps/dotcms-ui/src/app/view/components/dot-contentlet-editor/components/dot-contentlet-wrapper/dot-contentlet-wrapper.component.ts

```diff
@@ -42,8 +42,15 @@ export class DotContentletWrapperComponent {
     ) {
         if (!this.customEventsHandler) {
             this.customEventsHandler = {
-                close: () => {
+                close: ({ detail: { data } }: CustomEvent) => {
+                    debugger
                     this.onClose();
+                    if (data?.redirectUrl) {
+                        this.dotRouterService.goToEditPage({
+                            url: data.redirectUrl,
+                            language_id: data.languageId
+                        });
```

Comment:
I will normally ask for a test for this... but we are testing this in cypress :)

Reply:
I'll add a test on `dot-contentlet-wrapper.component.spec.ts`